src/libraries/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -257,4 +264,13 @@ public enum ProtocolErrors
         INADEQUATE_SECURITY = 0xc,
         HTTP_1_1_REQUIRED = 0xd
     }
+
+    public static class HttpVersion20
+    {
+#if !NETFRAMEWORK
+        public static readonly Version Value = HttpVersion.Version20;
```

Comment:
We don't compare for reference equality, so just the `#else` will work here too.